data/actions/scripts/quests/annihilator.lua

```diff
@@ -15,7 +15,7 @@ function onUse(player, item, fromPosition, target, toPosition, isHotkey)
 	if item.itemid == 1945 then
 		local players = {}
 		for _, position in ipairs(playerPosition) do
-			local topPlayer = Tile(position):getTopCreature()
+			local topPlayer = position:getTile():getTopCreature()
```

Comment:
Revert this, no need to go through another function when you can just use Tile(position).